src/pages/locationsettings/index.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Remove unused import
```suggestion
import React, { useState } from "react";